README.md

```diff
@@ -10,6 +10,9 @@ This library aims to make things a bit more comfortable, especially for containe
  - `psgo.ProcessInfo(descriptors []string) ([][]string, error)`
    - ProcessInfo returns the process information of all processes in the current mount namespace. The input descriptors must be a slice of supported AIX format descriptors in the normal form or in the code form, if supported.  If the input descriptor slice is empty, the `psgo.DefaultDescriptors` are used. The return value contains the string slice of process data, one per process.
 
+ - `psgo.ProcessInfoByPids(pids []string, descriptors []string) ([][]string, error)`
+   - ProcessInfoByPids returns the process information of a set of processes in the current mount namespace. The input pids must be a slice of PIDs. The input descriptors must be a slice of supported AIX format descriptors in the normal form or in the code form, if supported.  If the input descriptor slice is empty, the `psgo.DefaultDescriptors` are used. The return value contains the string slice of process data, one per process.
```

Comment:
I'd prefer to say that it's similar to `psgo.ProcessInfo` but will only return the data of the specified pids. It's less to read and easier to figure the differences among the functions.

Reply:
We also need to mention what happens when the slice is empty.

Reply:
When the slice is empty  a slice of size [1][n]  is returned containing the n format descriptor headers and no process information. That's ok or would you rather return nothing at all?

Reply:
Actually, there is another case when no matching pids could be found: "When the specified slice of pids is empty or no matching processes could be found, only the header without further data is returned."